src/meds_torch/predict.py

```diff
@@ -188,6 +219,51 @@ def predict(cfg: DictConfig, datamodule=None) -> tuple[dict[str, Any], dict[str,
         cfg: DictConfig configuration composed by Hydra.
     Returns:
         Tuple[dict, dict] with metrics and dict with all instantiated objects.
+
+    Examples:
+    >>> import tempfile
+    >>> from omegaconf import DictConfig
+    >>> _ = pl.Config.set_tbl_width_chars(106)
+    >>> # Create temporary checkpoint file
```

Comment:
you should mention in the docstring that you're using the dummy classes you created above.